code/game/objects/structures/window.dm

```diff
@@ -444,7 +444,7 @@
 /obj/structure/window/plasma/reinforced
 	name = "reinforced plasma window"
 	desc = "A window made out of a plasma-silicate alloy and a rod matrice. It looks hopelessly tough to break and is most likely nigh fireproof."
-	icon_state = "rplasmawindow"
+	icon_state = "plasmawindow"
```

Comment:
@MoreRobustThanYou Was the windoor location that was weird, I got the fulltile one fine.